Package.swift

```diff
@@ -53,12 +53,15 @@ let package = Package(
         ),
         .target(name: "OpenTracingShim",
                 dependencies: ["OpenTelemetrySdk",
-                               "Opentracing"]
+                               "Opentracing"],
+                path: "Sources/Importers/OpenTracingShim"
         ),
         .target(name: "SwiftMetricsShim",
```

Comment:
Should we also rename these to importers? `SwiftMetricsImporter` and `OpenTracingImporter`

Reply:
"Opentracing shim" name is used everywhere in the opentelemetry docs

Reply:
Fair enough, happy to leave - no worries 👍